### DISCUSSION TITLE
What's our ultimate goal?

### DISCUSSION THREAD

**Discussion by SimHoZebs** [ipod1g/battleship#16] [General]
What's the direction we are going with this? What are the skills you want to express through this project other than collaboration? Are we trying to one-up other battleship game sites? Do we want to be faithful to the original game?

Do you have a design philosophy in mind? Are we targeting simplicity and efficient design, or immersive, feature filled? What are some of the features you already have in mind?

**Comment by ipod1g** [comment]
Well, the initial purpose was quite simple.
1) Improve our collab skill, and in my case, programming abilities too
2) End product to be served as an experience/portfolio

and for the game...
I wanted it to be something we can casually play together (if it ends up to be fun) because when I played it in board game cafe, I thought it was fun and I wanted to add webcam to this game to simulate that experience where you just have fun with your friend.

And regarding design philosophy, I think my ideal goal is to have something scalable? Maybe the original battleship rule may soon turn out to be boring, and perhaps we could leave some room to give it a bit of a twist.

But that twist, I am not sure yet.
